e2e/support/helpers/e2e-search-helpers.js

```diff
@@ -48,7 +48,7 @@ export function expectSearchResultContent({
           });
         }
         if (expectedSearchResult.timestamp) {
-          cy.findByTestId("revision-history-button").findByText(
+          cy.findByTestId("revision-history-text").findByText(
```

Comment:
On search components, we don't pass any `onClick` prop, so it's rendered as text instead of a button as a part of the new change.